projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/json-editor-node-flat.component.html

```diff
@@ -29,7 +29,7 @@
       <div class="node-input">
         <!-- Number | Integer -->
         <div *ngIf="schema?.type === 'number' || schema?.type === 'integer'">
-          <ngx-input type="number" [ngModel]="model" (ngModelChange)="updateModel($event)" [requiredIndicator]="false"
+          <ngx-input type="number" [placeholder]="placeholder" [ngModel]="model" (ngModelChange)="updateModel($event)" [requiredIndicator]="false"
```

Comment:
As per our discussion, we don’t need to do this in the flat version. We will show examples in a tooltip.